src/common/wizard.js

```diff
@@ -0,0 +1,71 @@
+import _ from 'messages'
+import classNames from 'classnames'
+import Icon from 'icon'
+import map from 'lodash/map'
+import React, { Component, cloneElement } from 'react'
+
+import styles from './wizard.css'
+
+export default class Wizard extends Component {
+  render () {
+    const { children } = this.props
+    const allDone = children.every((child) => child.props.done || child.props.summary)
```

Comment:
Don't use array methods on `children`, it can also be `null` or a plain component, use `Chidren.toArray()` from `'react'` first.

And prefer lodash's methods over native ones.